### ISSUE TITLE
MySQLError.prepareStatus: Prepare response has invalid status

### ISSUE THREAD

**Issue by Grumme** [vapor/vapor#1569]
When I try to run the server, I get the following error when trying to migrate MySQL DB:

```
Migrating mysql DB
⚠️ [MySQLError.prepareStatus: Prepare response has invalid status] [/Users/jakobgrumsen/API/MeetingApp/.build/checkouts/mysql.git-1890032512239690518/Sources/MySQL/Protocol/MySQLComStmtPrepareOK.swift:25:122]
Program ended with exit code: 1
```

The only table which is created in the database is Fluent, which is empty.

### Steps to reproduce

I have been following the documentation: https://docs.vapor.codes/3.0/mysql/fluent/

Which means I have an user:

```
final class User: MySQLModel {
    /// The unique identifier for this user.
    var id: Int?
    
    /// The user's full name.
    var name: String
    
    /// The user's current age in years.
    var age: Int
    
    /// Creates a new user.
    init(id: Int? = nil, name: String, age: Int) {
        self.id = id
        self.name = name
        self.age = age
    }
}

extension User: Migration { }
```

Inside configure.swift I have the following:

```
try services.register(FluentMySQLProvider())

let mysqlConfig = MySQLDatabaseConfig(hostname: "localhost", port: 3306, username: "dbUsername", password: "dbPassword", database: "dbName")
services.register(mysqlConfig)

var migrations = MigrationConfig()
migrations.add(model: User.self, database: .mysql)
services.register(migrations)
```

My Package.resolved contains the following:

```
{
        "package": "Fluent",
        "repositoryURL": "https://github.com/vapor/fluent.git",
        "state": {
          "branch": null,
          "revision": "0307bdaee347ccb95411b11f0630959fef83c962",
          "version": "3.0.0-rc.2.1.2"
        }
      },
      {
        "package": "FluentMySQL",
        "repositoryURL": "https://github.com/vapor/fluent-mysql.git",
        "state": {
          "branch": null,
          "revision": "6f577efbc369b4c0e54a0edbd400af18aa30844e",
          "version": "3.0.0-rc.2.1"
        }
      },
      {
        "package": "MySQL",
        "repositoryURL": "https://github.com/vapor/mysql.git",
        "state": {
          "branch": null,
          "revision": "b27f3abb333dd4a5980caed8635ea654050d3e0e",
          "version": "3.0.0-rc.2.1.1"
        }
}
```

### Environment

* Vapor Framework version: 3.0.0-rc.2.0.2
* Vapor Toolbox version: 3.1.4
* OS version: 10.13.3

**Comment by tanner0101** [comment]
What MySQL version are you using?

**Comment by Mladen-K** [comment]
5.7 on AWS, with the default settings. I’m not sure of the latest minor version, but will check it out.

**Comment by Mladen-K** [comment]
It is 5.6.37 actually. I checked it. Sorry for the misinformation.

**Comment by tanner0101** [comment]
This will be fixed with the official 3.0.0 tag, we have code working for 5.0..<5.7 (still need to test 8.0 haha). Closing here since it's not related to vapor/vapor.
